### ISSUE TITLE
Image Picker From Google Drive Issue

### ISSUE THREAD

**Issue by mikemike396** [alhazmy13/MediaPicker#51]
So I have been getting come crashes related to the URI being not found. I found this to happen when using the Image Picker and selecting a non image type file (ZIP, MP4). Whats odd is the onActivityResult returns a URI back but its not valid it contains a .jpg extension and returns the exception below when you try to open it into a input stream. Anyway to restrict google drive selection to photos only? Or is there a way to disable showing third party apps and just show the built in gallery app?

Thank You

```
Exception (I picked a .mp4 file from my google drive account):
java.io.FileNotFoundException: /storage/emulated/0/mediapicker/images/ad220b5b-f716-42ae-ac6b-002879870f0f.jpg: open failed: ENOENT (No such file or directory)
```

![device-2017-12-21-112158](https://user-images.githubusercontent.com/14999806/34264765-e7740704-e641-11e7-9e88-c056ed635c97.png)

**Comment by alhazmy13** [comment]
Hi @mikemike396 
If you want to disable third-party apps like Google Drive then just add this option to your builder `.allowOnlineImages(false)`.
I'm wondering why you can select zip files with google drive; it supposes to be disabled. Anyway, I'll take a look and raise it as a bug.
Can you please tell me what kind of devices you are using? Is it a real device or emulator?
Thanks

**Comment by mikemike396** [comment]
This was a real device. Samsung Galaxy Note 3 - Android 5.0

**Comment by mikemike396** [comment]
Any update on this?

**Comment by alhazmy13** [comment]
Fixed with version [2.4.0](https://github.com/alhazmy13/MediaPicker/releases/tag/2.4.0)
